.travis.yml

```diff
@@ -29,16 +29,17 @@ addons: &addons
         #  branch_pattern: master
 
 before_script:
-  - export CFLAGS="-W -Wall -O2 -g -I/home/travis/build/argorain/cmocka/include/"
-  - export LDFLAGS="-L/home/travis/build/argorain/cmocka/lib/"
+  - export CFLAGS="-W -Wall -O2 -g -I~/cmocka/include/"
```

Comment:
`~` is bashism which is not interpolated in double quotes. Use either $HOME or full path.